### PR TITLE
Update AboutUs page

### PR DESCRIPTION
**Pull request by muraddurrani**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -1,7 +1,4 @@
----
-layout: page
-title: About Us
----
```
Comment (ckcherry23) [inline docs/AboutUs.md:4]: I think this is required to follow the layout of the website

File: docs/AboutUs.md [hunk 2]
```diff
@@ -1,7 +1,4 @@
...
+#About Us
```
Comment (ckcherry23) [inline docs/AboutUs.md:1]: This can be removed as layout handles the title

File: docs/team/muraddurrani.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#Murad Durrani's Project Portfolio Page
```
Comment (ckcherry23) [inline docs/team/muraddurrani.md:1]: Can add the layout table here as well

**Review by ckcherry23 — CHANGES_REQUESTED** [review]

**Review by tonishka — APPROVED** [review]

**Review by ckcherry23 — APPROVED** [review]
LGTM

**Pull request merged**